### DISCUSSION TITLE
Saving Precomputing data

### DISCUSSION THREAD

**Discussion by ralphgeorg-s** [libigl/libigl#2024] [Q&A]
Hi, for the Biharmonic Deformation the precomputation takes a while. We would like to pre-compute, and save the data. Can you pl. advice on how to save the data from the memory in a JSON and use it in real time?

**Comment by alecjacobson** [comment]
The major part of the precomputation is storing a "factorization" of a sparse matrix. This is an Eigen object which I'm not entirely familiar with. In theory it should just but a few (2 or 3) sparse matrices. I'm guessing that these can be easily serialized, but it's a bit more of a question for Eigen. If the factorizations are stored as typical Eigen Sparse matrices, then you could just write out the compressed column/row storage as three arrays.
